base/interactiveutil.jl

```diff
@@ -64,7 +64,7 @@ function edit(path::AbstractString, line::Integer=0)
     nothing
 end
 
-function edit(m::Method)
+function edit(m::TypeMapEntry)
```

Comment:
the second field could be removed, but it would require the creation of expensive guard entries (e.g. with tvar environments), so i don't think the performance/memory trade-off there makes sense

> I think the fastest way to fix this is to put a sig field back in Method

also seems acceptable. having a duplicate copy of these fields doesn't seem like the worst thing ever (I don't think it'll be noticeable for performance or memory)